src/modules/commands/duel.js

```diff
@@ -0,0 +1,36 @@
+const isEmpty = require('lodash/isEmpty');
+const command = require('../command.filter');
+
+const duelling = async function (request, context) {
+    const { i18n, send, getUser } = context;
+    const { user, args: userId } = request;
+
+    const user1 = user;
+    const user2 = await getUser(userId);
+
+    if (isEmpty(user2)) {
+        send(i18n('duel.userisnotdefined', { id: user1 }));
+
+        return request;
+    }
+
+    if (user2 === user1) {
+        send(i18n('duel.cantduelauthor', { id: user1 }));
+
+        return request;
+    }
+
+    const users = [];
```

Comment:
зачем ниже делать push если ты сразу можешь сделать
```suggestion
    const users = [user1, user2];
```

Reply:
Кроме этого `await`ты там не нужны

Reply:
угу, push это sync операция :)